pcs/lib/corosync/live.py

```diff
@@ -37,7 +36,7 @@ def set_expected_votes(runner: CommandRunner, votes: int) -> str:
     """
     stdout, stderr, retval = runner.run(
         [
-            os.path.join(settings.corosync_binaries, "corosync-quorumtool"),
+            settings.corosync_quorumtool_exec,
             "-e",
             str(votes),
```

Comment:
```suggestion
            str(votes)
```
This will make `black` to put the whole list in one line. It's a formatting change, but it's small enough for us to take it.